passes/cmds/splitnets.cc

```diff
@@ -226,7 +226,7 @@ struct SplitnetsPass : public Pass {
 
 					vector<IdString> &new_ports = module_ports_db[module->name][wire->name];
 
-					for (SigSpec c : sig.chunks())
+					for (const SigSpec &c : sig.chunks())
```

Comment:
So `sig.chunks()` returns a reference to a `std::vector<RTLIL::SigChunk>` we then iterate over that list and convert each element into a `SigSpec`. In what way is it an improvement to then create a reference to that freshly created SigSpec? This is not avoiding any copy, it's just adding an additional indirection.

Reply:
Again, in this case there will not be any change in the code output (it is like assigning a local variable to a const reference). Here as well, this will only be for readability improvement.